src/lib/stores/user.ts

```diff
@@ -40,12 +46,21 @@ function createUserStore() {
 	 */
 	const saveGame = async (slug: string): Promise<boolean> => {
 		const success = await saveInCache(slug);
+
 		if (success) {
-			savedGames.add(slug);
-			update((user) => {
-				user.savedGames = [...savedGames];
-				return user;
+			const sync = await fetch('/api/user/gymles/add.json', {
+				method: 'POST',
+				body: JSON.stringify([slug])
 			});
+			const body = await sync.json();
+
+			if (sync.ok && Array.isArray(body) && body.includes(slug)) {
+				console.log('Game synced with DB', body, slug);
```

Comment:
Gewoon uit interesse. Zijn al die console logs nodig? Staan op meer plekken. Handig voor debugging, maar wil je dat in live versie hebben. Ook die `activated` van SW zeg maar

Reply:
Ik vind het wel chill, om een klein beetje feedback te hebben in de console. Zeker op een live versie, dan hoef je niet eerst een push te doen om de logs erin te zetten. Ze staan nu nog wat arbitraire.

Liefste had ik een log module gemaakt. 

Ik kan ipv `log` ook `debug` gebruiken, die komt niet standaard in de console te staan..

Reply:
Ik heb het nu veranderd naar `debug`